rasa/core/evaluation/marker_stats.py

```diff
@@ -12,12 +12,15 @@
 
 def compute_statistics(
     values: List[Union[float, int]]
-) -> Dict[Text, Union[int, np.float]]:
+) -> Dict[Text, Union[int, float]]:
     """Computes some statistics over the given numbers."""
     return {
         "count": len(values) if values else 0,
         "mean": np.mean(values) if values else np.nan,
-        "median": np.median(values) if values else np.nan,
+        # [numpy-upgrade] type ignore can be removed after upgrading to numpy 1.23
```

Comment:
There's many of these untyped function call mypy errors in 1.21. I tried 1.23 and they disappeared again, that's why I think it's okay to temporarily ignore them and then once we upgrade we can remove it again. Upgrading to >1.22, however, would also result in losing python 3.7 support